test/jdk/java/awt/print/PrinterJob/ImagePrinting/NullClipARGB.java

```diff
@@ -26,27 +26,36 @@
  * @key printer
  * @bug 8061392
  * @summary Test no NPE when printing transparency with null clip.
+ * @library /test/lib
  */
 
 import java.awt.*;
 import java.awt.image.*;
 import java.awt.print.*;
+import javax.print.DocFlavor;
+import javax.print.PrintService;
+import javax.print.PrintServiceLookup;
+import jtreg.SkippedException;
 
 public class NullClipARGB implements Printable {
 
     public static void main( String[] args ) {
+        PrintService[] svc = PrintServiceLookup.lookupPrintServices(DocFlavor.SERVICE_FORMATTED.PRINTABLE, null);
+        if (svc.length == 0) {
+            throw new SkippedException("Printer is required for this test.  TEST ABORTED");
+        }
 
         try {
             PrinterJob pj = PrinterJob.getPrinterJob();
             pj.setPrintable(new NullClipARGB());
             pj.print();
-            } catch (Exception ex) {
```

Comment:
I see this is a cleanup, but we don't need to do it here, to keep patch simple and backportable.